src/api/services/action.py

```diff
@@ -28,7 +28,7 @@ def list_actions(self, context: Context, args) -> Tuple[list, MassEnergizeAPIErr
     actions, err = self.store.list_actions(context, args)
     if err:
       return None, err
-    return serialize_all(actions), None
+    return actions, None
```

Comment:
What you've done is fine, but I think it would be slightly cleaner to have the pagination in the service layer routines.
So instead of:
return serialize_all(actions) 
replace with
return paginate(actions,page,limit)

Reply:
I will clean it up